Fiyiin/TL-QA-selector
# #164802781 Create Index and team Page

### What does this PR do?
This PR creates the UI for team and index page

### Description of Task to be completed
create the user interface using bootstrap

### How should this be manually tested?
clone the repo
checkout to ft-UI branch
check index and team page using live server

### What are the relevant pivotal tracker stories?
- #164802781
